ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/uniqueness/model/UniquenessCheckTransactionDetails.kt

```diff
@@ -0,0 +1,12 @@
+package net.corda.v5.ledger.utxo.uniqueness.model
+
+import net.corda.v5.crypto.SecureHash
+
+/**
+ * Representation of the transaction details that must be persisted by the uniqueness checker,
+ * used by the uniqueness checker and backing store.
```

Comment:
Should also mention usage by the client?

Reply:
Actually, I just realised that this class is not used in the API at all, so can be moved to the runtime-os.